doc/services/device_mgmt/smp_groups/smp_group_10.rst

```diff
@@ -0,0 +1,330 @@
+.. _mcumgr_smp_group_10:
+
+Enumeration Management Group
+#############################
+
+Enumeration management group defines the following commands:
+
+.. table::
+    :align: center
+
+    +----------------+-----------------------------+
+    | ``Command ID`` | Command description         |
+    +================+=============================+
+    | ``0``          | Count of supported groups   |
+    +----------------+-----------------------------+
+    | ``1``          | List supported groups       |
```

Comment:
Note for the future: command to enumerate lists one by one should be also available. List is ok, for clients that can freely allocate memory for the list, but in case of smaller devices it may be easier to enumerate, using the counter, all the registered groups.